deploy/ansible/appsmith_playbook/main.yml

```
@@ -0,0 +1,60 @@
+---
+  - name: Configure the self-hosted server
+    hosts: appsmith
+    any_errors_fatal: true
+    vars:
+        integromat_webhook_uri: https://hook.integromat.com/dkwb6i52am93pi30ojeboktvj32iw0fa
```

```suggestion
        analytics_webhook_uri: https://hook.integromat.com/dkwb6i52am93pi30ojeboktvj32iw0fa
```

Minor suggestion to rename the variable to `analytics_webhook_uri`. Ensures that we can change the analytics endpoint without affecting the variable name.